### DISCUSSION TITLE
Need more volunteers

### DISCUSSION THREAD

**Discussion by soufi** [mulesoft-anypoint/terraform-provider-anypoint#4] [General]
Hi, 

Being the main maintainer of this project, It's been hard for me to find time to work on this project. 

Moreover, this fork is a workaround to a multitude of issue we've encountered along the way due to the lack of support from Mulesoft technical teams (Github especially). 
I am no longer a Mulesoft employee therefore I can't push to the main repository mulesoft-consulting/terraform-provider-anypoint. 
Instead I will be doing all my work here. I'll send P.Rs to the main repository, but it's going to be hard to find people that will be able to validate the PRs ...

Now, I'm looking for people who are willing to spend some time for the community and help with organisation and delivery of new features. 

Cheers

**Comment by tools400** [comment]
Is there an easy to follow documentation how to set up a local test environment with the [anypoint-client-go](https://github.com/mulesoft-anypoint/anypoint-client-go) and the [terraform-provider-anypoint](https://github.com/mulesoft-anypoint/terraform-provider-anypoint)?

I can build the Terraform provider and override the provider to use on my local PC (`.terraformrc` / `terraform.rc`). But I do not know how to debug both projects at a time and how to include a beta version of the **anypoint-client-go** into a beta version of the **terraform-provider-anypoint** project.
